app/ui/pages/10_Settings.py

```diff
@@ -82,6 +86,14 @@ def task_listing(
             key="serverless_credit_cost",
         )
         storage = st.text_input("Storage Cost (/tb)", value=tbcost, key="storage_cost")
+        timezone = st.selectbox(
+            "Default Timezone",
+            index=next(
+                i for i, v in enumerate(pytz.all_timezones) if v == timezone_start
```

Comment:
nit: Was glancing at `all_timezones` and see that it has over 500 entries in it. Would `common_timezones` be abrasive?

Reply:
fixed to use common_timezones. Makes a lot more sense